maintenance_curation.Rmd

```diff
@@ -0,0 +1,238 @@
+    package.
+
+## Peer-reviewed packages
+
+Peer-reviewed packages are those contributed to the rOpenSci by the
+community and have passed through peer review. They need to be
+[in-scope](#aims-and-scope)
+at the time of submission to be reviewed.
+
+-   Upon acceptance, these peer-reviewed packages are transferred from
+    the author's GitHub to the "ropensci" GitHub organization
+
+-   Peer-reviewed packages will be in the registry tagged as
+    "peer-reviewed" and have a peer-reviewed badge in their README.
+
+-   Peer-reviewed packages will be listed on rOpenSci's web page or
+    similar venues with tag "peer-reviewed"
+
+-   Peer-reviewed packages will be checked via rOpenSci's CI that they
+    build and test successfully across systems and that their docs
+    build correctly.
+
+-   When packages fail on CI, rOpenSci's build system will notify the
+    authors automatically. If on CRAN, it will also notify them of
+    additional issues raised, opting peer-reviewed package authors in
+    to this service by default (see [issue](https://github.com/ropenscilabs/cchecksapi/issues/72)).
+
+-   Annually or bi-annually, rOpenSci staff will review packages in a
+    failing state or have been failing for extended periods, and
+    contact the authors to determine ongoing maintenance status and
+    expected updates. Based on this exchange, rOpenSci may opt to
+    retain the package's current status with the expectation of an
+    updates, contribute support or seek a new maintainer, or transfer
+    the package to "archived" status.
+
+-   Based on user base (measured by downloads), reverse dependencies, or
+    rOpenSci strategic goals, rOpenSci staff may support failing
+    packages via PRs reviewed by package authors, or direct changes
+    (if authors are unresponsive for approximately a month). rOpenSci
+    will also provide support to package authors on request, both by
+    staff and community volunteers, based on time available.
+
+-   At the author's request, or if authors are unresponsive to
+    inquiries for approximately a month, rOpenSci may seek a new
+    maintainer for select peer-reviewed packages it deems have high
+    community value based on user base/downloads, reverse
+    dependencies, or rOpenSci strategic goals.
+
+-   When archived, these packages will move from the "ropensci" GitHub
+    organization to the "ropensci-archive" organization (or author
+    GitHub accounts if desired). They will gain the "archived" tag
+    in the registry, and a "migrated" tag and "migrated-date"
+    field indicating the date of migration and a URL for a new
+    location if needed.. They will retain "peer-reviewed" tags and
+    badges. They will not be checked via CI.
+
+-   Archived packages will not be displayed by default. An option to
+    "show archived packages" will display these with an "archived"
+    tag.
+
+-   Archived packages may be un-archived at authors' request if authors
```

Comment:
and at the request of new people?
same question as for staff maintained packages: then they go back to ropensci-incubator?